src/components/FacilitiesPageViewMore/DeleteFacility.jsx

```diff
@@ -1,92 +1,90 @@
-              <ModalFooter>
-                <Button
-                  className="cancelButton"
-                  width="250px"
-                  size="sm"
-                  color="--cds-blue-2"
-                  variant="outline"
-                  onClick={onClose}
-                >
-                  Cancel
-                </Button>
-                <ButtonGroup variant="outline" spacing="6">
-                  <Button
-                    className="deleteButton"
-                    width="250px"
-                    size="sm"
-                    bg="#21307a"
-                    color="white"
-                    onClick={() => handleConfirmDelete(props.id)}
-                  >
-                    Yes, remove the facility
-                  </Button>
-                </ButtonGroup>
-              </ModalFooter>
-            </ModalContent>
-          </Modal>
-        );
-    }
-    
-    const handleConfirmDelete = async id => {
+  const { isOpen, onOpen, onClose } = useDisclosure();
+  const { backend } = useBackend();
+  function ShowModal({ isOpen, onClose }) {
+    return (
+      <Modal isOpen={isOpen} onClose={onClose}>
+        <ModalOverlay />
+        <ModalContent>
+          <ModalHeader>Remove Facility</ModalHeader>
+          <ModalCloseButton />
+          <ModalBody>
+            Are you sure you want to remove the facility from the adoption log? Once you delete
+            them, there is no way of getting the information back
+          </ModalBody>
+
+          <ModalFooter>
+            <Button
+              className="cancelButton"
+              width="250px"
+              size="sm"
+              color="--cds-blue-2"
+              variant="outline"
+              onClick={onClose}
+            >
+              Cancel
+            </Button>
+            <ButtonGroup variant="outline" spacing="6">
+              <Button
+                className="deleteButton"
+                width="250px"
+                size="sm"
+                bg="#21307a"
+                color="white"
+                onClick={() => handleConfirmDelete(props.id)}
+              >
+                Yes, remove the facility
+              </Button>
+            </ButtonGroup>
+          </ModalFooter>
+        </ModalContent>
+      </Modal>
+    );
+  }
+
+  const handleConfirmDelete = async id => {
     try {
-        console.log(`Deleted facility with id ${id}`);
-        const response = await backend.delete(`/facility/${id}`);
+      console.log(`Deleted facility with id ${id}`);
+      const response = await backend.delete(`/facility/${id}`);
 
-        onClose();
+      onClose();
 
-        if (response.status === 200) {
-            CreateToast({
-                description: `${facilityName} deleted successfully`,
-                status: 'success',
-                toast,
-            });
-        }
+      if (response.status === 200) {
+        CreateToast({
+          description: `${facilityName} deleted successfully`,
```

Comment:
🚫 **[eslint]** <[no-undef](https://eslint.org/docs/rules/no-undef)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'facilityName' is not defined.